_pages/en/open-source-code-form-validation.html

```diff
@@ -0,0 +1,252 @@
+---
+layout: default
+ref: formValidation
+lang: en
+permalink: /en/open-source-code-form-validation.html
+---
+
+<h1 property="name" id="wb-cont">Form with validation for open source code</h1>
+<div class="wb-frmvld">
+<form action="#" method="get" id="validation">
+
+<legend><span class="field-name"> Version of the metadata schema</span></legend>
+	<div class="form-group">
+		<label for="shemaVersion" class="required"><span class="field-name">schemaVersion</span> <strong class="required">(required)</strong></label>
+			<select class="form-control" id="shemaVersion" name="shemaVersion" required="required">
```

Comment:
To make it easy, schemaVersion should just default to 1.  We don't need to have the dorpdown to select the version.

Add a link to the metadata schema page